src/App.jsx

```diff
@@ -5,6 +5,8 @@ import HomePage from './pages/HomePage';
 import ProductListing from './pages/ProductListing';
 import CartPage from './pages/CartPage';
 import WishlistPage from './pages/WishlistPage';
+import Login from './pages/Login';
```

Comment:
can make an index page and import from there looks more clean

Reply:
Okay, I'll make the changes!